### DISCUSSION TITLE
Question: error using convert-di-to-inject

### DISCUSSION THREAD

**Discussion by kscavenius** [nartc/ngxtension-platform#399] [Q&A] [answered]
Hi

I am experimenting with the great convert-di-to-inject functionality. When running **ng g ngxtension:convert-di-to-inject** i get the error:
![image](https://github.com/ngxtension/ngxtension-platform/assets/18103936/a10503d8-6fa0-41ab-9a43-6bd42ab1d711)

My app compiles just fine, and it works great when specifying a component to migrate. 

All the best
Kristoffer

**Comment by eneajaho** [comment]
Hello @kscavenius 
We need more information in order to solve your issue. Can you provide a small repro please?

**Comment by kscavenius** [comment]
Hey Enea

I have been away from the code in the last week. Apologies.

Since it works when I run it on specific components individually, and only fails when running it on the whole repository, it is probably an issue related to a specific component. I cannot share the project as a whole, but I can share the specific component .ts file. Should I do that?

I am new to GitHub collaboration, so bear with me 😳

Best regards
Kristoffer

Sent from [Proton Mail](https://proton.me/mail/home) for iOS

På fre., 14 jun., 2024 kl. 12:45, Enea Jahollari ***@***.***(mailto:På fre., 14 jun., 2024 kl. 12:45, Enea Jahollari <<a href=)> skrev:

**Reply by eneajaho** [reply, marked as answer]
Hello @kscavenius 
Can you try running the official Angular migration schematic for this: 
https://github.com/angular/angular/pull/57056 ?
It was released in https://github.com/angular/angular/releases/tag/18.2.0-next.2

Closing the discussion because we will stop supporting the migration schematic and recommend to use the official one from now on.

**Comment by kscavenius** [comment]
Hi Enea

The official migration appears to be working just fine, thanks.

/Kristoffer

Sent with [Proton Mail](https://proton.me/) secure email.
